File: Android/app/src/main/java/catglo/com/deliveryDatabase/Shift.java
package catglo.com.deliveryDatabase;


import org.joda.time.MutableDateTime;

public class Shift {
	public MutableDateTime startTime;
	public MutableDateTime endTime;
	public int odometerAtShiftStart;
	public int odometerAtShiftEnd;
	public int primaryKey;
	public boolean noEndTime=false;
	public Shift(){
		startTime = new MutableDateTime(MutableDateTime.now());
		endTime   = new MutableDateTime(MutableDateTime.now());
	}
}

File: Android/app/src/main/java/catglo/com/deliveryDatabase/Shift.kt
package catglo.com.deliveryDatabase


import org.joda.time.MutableDateTime

class Shift {
    var startTime = MutableDateTime(MutableDateTime.now())
    var endTime = MutableDateTime(MutableDateTime.now())
    var odometerAtShiftStart: Int = 0
    var odometerAtShiftEnd: Int = 0
    var primaryKey: Int = 0
    var noEndTime = false
}
